bigml/logistic.php

```diff
@@ -171,16 +173,22 @@ public function __construct($logistic_regression, $api=null, $storage="storage")
 
             $this->coefficients=array();
 	    if (property_exists($logistic_regression_info, "coefficients")) {
+              $j=0;
 	      foreach ($logistic_regression_info->coefficients as $key => $coefficient) {
 	         $this->coefficients[$coefficient[0]] = $coefficient[1];
+                 if ($j == 0 and !is_array($coefficient[1])) {
+                    $old_coefficients=true;
+                 }
+                 $j+=1; 
 	      } 
 	    }
-
+ 
             $this->bias = property_exists($logistic_regression_info, "bias") ? $logistic_regression_info->bias : 0;
```

Comment:
again, long lines. I leave it to you to consider whether this is acceptable coding style in PHP... (I am not going to highlight more cases of long lines)